enzy_htp/chemical/residue.py

```diff
@@ -33,46 +230,5916 @@
 
 RESIDUE_CONNECTIVITY_MAP: Dict[str,
                                Dict[str,
-                                    List[str]]] = load_from_db("RESIDUE_CONNECTIVITY_MAP")
+                                    List[str]]] = {
+        "ALA": {
```

Comment:
Is it possible to not make it so long? the original format looks fine to me

Reply:
Oh just found out it was like this in the json. Maybe you can refer to AmberMaps

Reply:
Updated this